### ISSUE TITLE
B_박성빈_lab_week2

### ISSUE THREAD

**Issue by 03psb** [vlab-kaist/NeuralNet101_2022#21]
### LAB 3 - 박성빈

**Problem 1**

'''
    import torch
    import pandas as pd

    df = pd.read_excel("delivery_time.csv")
    x_train = torch.tensor(df['Sorting_Time'].values.tolist(), dtype=torch.float32)
    y_train = torch.tensor(df['Delivery_Time'].values.tolist(), dtype=torch.float32)

    W = torch.zeros(1, requires_grad=True, dtype=torch.float32)
    b = torch.zeros(1, requires_grad=True, dtype=torch.float32)

    optimizer = torch.optim.SGD([W, b], lr=0.01)

    nb_epochs = 5000

    for epoch in range(nb_epochs):
        hypothesis = x_train * W + b
        cost = torch.mean((hypothesis - y_train) ** 2)

        optimizer.zero_grad() # initialize gradient
        cost.backward() #calculate gradient
        optimizer.step() #change W, b

    x_predict = torch.FloatTensor([11])
    y_predict = x_predict * W + b

    print("W: {0}".format(W))
    print("b: {0}".format(b))
    print("loss function cost: {0}".format(cost))
    print("Predicted Delivery_Time when Sorting_Time is 11: {0}".format(y_predict))
'''

'''
    W: tensor([1.6490], requires_grad=True)
    b: tensor([6.5826], requires_grad=True)
    loss function cost: 7.793311595916748
    Predicted Delivery_Time when Sorting_Time is 11: tensor([24.7220], grad_fn=<AddBackward0>)
'''
